pages/ksphere/kaptain/1.0.1-0.5.0/introducing/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: layout.pug
+navigationTitle: Introducing Kaptain
+title: Introducing Kaptain
+excerpt: Deploy and manage machine learning models with ease
+menuWeight: 1
+---
+Kaptain is a cloud-native suite of best-of-breed open-source technologies that allow data scientists to extract value from data immediately by providing a familiar environment for development and all the technologies needed to deploy and scale models in production. Kaptain solves a key problem that enterprises face: How to get a return from your expensive AI investments? Promoting from prototype to production is often hard, but it does not have to this case, with Kaptain.
+
+D2iQ's Kaptain leverages our Expertise Kubernetes, Konvoy, so companies can run their machine learning workloads anywhere: in the cloud, on-premise, or in hybrid environments.
```

Comment:
I don't really understand what 'our Expertise Kubernetes, Konvoy' is supposed to mean.

Is it meant to mean "leverages our expertise, Kubernetes, as expressed in our product, Konvoy"?  

Here's my suggestion:

```suggestion
D2iQ's Kaptain leverages our expertise in Kubernetes, so that companies can run their machine learning workloads anywhere: in the cloud, on-premise, or in hybrid environments.
```